tools/cli/persistenceUtil.go

```diff
@@ -73,6 +73,17 @@ func CreatePersistenceFactory(c *cli.Context) persistenceClient.Factory {
 func CreateDefaultDBConfig(c *cli.Context) (config.DataStore, error) {
 	engine := getRequiredOption(c, FlagDBEngine)
 
+	var tls auth.TLS
```

Comment:
you probably want to make this a pointer, right? especially since the defaultConfig.TLS value is a pointer and a null value is probably preferable to an empty struct

Reply:
done